README.md

```diff
@@ -89,24 +89,24 @@ Try saying e.g. `air bat cap` to insert abc.
 
 All key commands are defined in [keys.talon](core/keys/keys.talon). Say letters of the [Talon alphabet](#alphabet) for A–Z.
 
-For modifier keys, say `help modifiers`. For example, say `shift air` to press `shift-a`, which types a capital `A`.
+For modifier keys, say `help modifiers`. For example, say `shift air` to press <kbd><kbd>shift</kbd>+<kbd>a</kbd></kbd>, which types a capital `A`.
```

Comment:
```suggestion
For modifier keys, say `help modifiers`. For example, say `shift air` to press <kbd>Shift</kbd> + <kbd>A</kbd>, which types a capital `A`.
```
The above with a space on each side of the + and no nested `<kbd>` is more readable: <kbd>Shift</kbd> + <kbd>A</kbd>. Also if we're using Windows conventions then modifiers and alphabetic keys should be capitalized.